### PR TITLE
test: ✅ fix codesandbox ci

### PR DESCRIPTION
**Pull request by afc163**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Perfermance optimization
- [ ] Refactoring
- [ ] Code style optimization
- [x] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 🔗 Related issue link

https://github.com/ant-design/ant-design/pull/24685#issuecomment-637334917

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |           |
| 🇨🇳 Chinese |           |

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -47,7 +47,7 @@
   "scripts": {
     "api-collection": "antd-tools run api-collection",
     "authors": "git log --format='%aN <%aE>' | sort -u | grep -v 'users.noreply.github.com' | grep -v 'gitter.im' | grep -v '.local>' | grep -v 'alibaba-inc.com' | grep -v 'alipay.com' | grep -v 'taobao.com' > AUTHORS.txt",
-    "build": "npm run compile && npm run dist",
+    "build": "npm run compile && NODE_OPTIONS='--max-old-space-size=4096' npm run dist",
```
Comment (yoyo837) [inline package.json:50]: 不够可以再调大点，根据主机物理内存大小来调看看能不能好。

Comment (xrkffgg) [reply package.json:50]: 是内存的原因吗？

**Comment by afc163** [comment]
成功了。

image

**Comment by hengkx** [comment]
image
dist有点大啊

**Comment by yoyo837** [comment]
原来`NODE_OPTIONS`还可以这样用 http://nodejs.cn/api/cli/node_options_options.html

**Pull request merged**